### ISSUE TITLE
[FEATURE] Closed Chain Support

### ISSUE THREAD

**Issue by Jad-Tawil** [engnadeau/pybotics#888]
Hello,

Does this library support closed chain robots, such as the parallel delta robot?

**Comment by engnadeau** [comment]
yes, if you model each chain independently and then put kinematic constraints on the optimizations (e.g., for calibration)
